WordPress/gutenberg
# Update to Babel 7.4 and core-js 3

## Description

Specifies the version of `core-js` used in the package (Gutenberg uses 2.x at the moment) to prevent warnings.

## How has this been tested?

1. Install `wp-scripts` in your project.
2. Try to run some tests
3. Notice that there's no error anymore.

## Screenshots 

## Types of changes
Bug fix (non-breaking change which fixes an issue)

## Checklist:

Fixes #15138.
